spec/big_brother/app_spec.rb

```diff
@@ -45,7 +45,7 @@ def tick
         last_response.body.should =~ /^Running: false$/
       end
 
-      it "returns 'Running: true' and the combined weight when the cluster is running" do
+      it "returns 'Running: true', the number of backends, and the combined weight when the cluster is running" do
```

Comment:
Perhaps `the number of backends (healthy and live)` as it returns both?

Reply:
Sounds good, I'll improve the wording on the test descriptions.